pkg/prometheus/operator.go

```diff
@@ -2157,3 +2163,11 @@ func validateRelabelConfig(rc monitoringv1.RelabelConfig) error {
 	}
 	return nil
 }
+
+func validateProberURL(url string) bool {
+	if strings.Contains(url, ":") {
+		hostPort := strings.Split(url, ":")
+		return govalidator.IsHost(hostPort[0]) && govalidator.IsPort(hostPort[1])
```

Comment:
Can `hostPort[0]` be an IP. If so, we might need something like 
```
validHost := govalidator.IsHost(hostPort[0]) || govalidator.IsIP(hostPort[0])
return validHost && govalidator.IsPort(hostPort[1])
```

Reply:
ya good point :)

Reply:
ok I double checked the [IsHost() method](https://github.com/asaskevich/govalidator/blob/f21760c49a8d602d863493de796926d2a5c1138d/validator.go#L851-L853 ), it checks if input is either dnsname or IP. I just need IsHost() method then I can remove IsIP() from validate function itself